UnityProject/Assets/Scripts/Systems/ChemistryInMainAssembly/ChemistryManager.cs

```diff
@@ -10,6 +10,10 @@ public class ChemistryManager : MonoBehaviour
 	private static bool generatedReferences = false;
 	public void Awake()
 	{
+#if UNITY_EDITOR
```

Comment:
if possible, avoid using preprocessors. For this case you use ``if (application.isEdior)``.

Also, what is the reason to force it to false when in editor?

Reply:
so, it regenerates every time you start the game

Reply:
I don't really see the issue with using teh preprocessors